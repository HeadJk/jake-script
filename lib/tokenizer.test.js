"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const tokenizer_1 = require("./tokenizer");
test('it should pass', () => {
    expect((0, tokenizer_1.tokenizer)('log(1+3+5)')).toMatchSnapshot();
});
