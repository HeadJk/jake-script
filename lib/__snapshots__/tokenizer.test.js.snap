// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should pass 1`] = `
Array [
  Object {
    "kind": "IdentifierToken",
    "value": "log",
  },
  Object {
    "kind": "OpenParenToken",
  },
  Object {
    "kind": "NumericLiteralToken",
    "value": "1",
  },
  Object {
    "kind": "PlusToken",
  },
  Object {
    "kind": "NumericLiteralToken",
    "value": "3",
  },
  Object {
    "kind": "PlusToken",
  },
  Object {
    "kind": "NumericLiteralToken",
    "value": "5",
  },
  Object {
    "kind": "CloseParenToken",
  },
]
`;
